app/views/posts/index.html.haml

```diff
@@ -0,0 +1,13 @@
+.container{:style => "font-family: Open Sans; text-align: center;"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace